Makefile

```diff
@@ -213,15 +213,15 @@ compile_cpp = $(CMD_PREFIX)mkdir -p `dirname $(1)`; \
 	     $(CPP) $(CPPFLAGS) -x c $(2) | grep -v "\#" > $(1)
 compile_cc_dep = $(CMD_PREFIX)mkdir -p `dirname $(1)`; \
 	     echo " CC-DEP    $(subst $(build_dir)/,,$(1))"; \
-	     echo `dirname $(1)`/ \\  > $(1) && \
+	     printf %s `dirname $(1)`/  > $(1) && \
```

Comment:
More on quotes: does the `dirname $(1)` need to be double quoted here ? Can there be a problem if a path component name has a space ?

Reply:
Then the whole makefile collapses.

Reply:
Fair enough